app/index.html

```diff
@@ -27,8 +24,27 @@
 <link rel="icon" href="img/icons/32.png" sizes="32x32">
 <link rel="icon" href="img/icons/512.png" sizes="512x512">
 <link rel="icon" href="img/icon.svg" sizes="any" type="image/svg+xml">
+<meta name="application-name" content="Project Link">
+<meta name="format-detection" content="telephone=no">
+<meta name="msapplication-tap-highlight" content="no">
+<meta name="msapplication-TileColor" content="#00aba9">
+<meta name="msapplication-TileImage" content="img/static-icons/mstile-144x144.png">
+<meta name="apple-mobile-web-app-capable" content="no">
```

Comment:
I set `apple-mobile-web-app-capable` to `no` for now while we make the app works on environments with no IndexedDB support.